src/styles/base/_theme-css-vars.scss

```
@@ -0,0 +1,74 @@
+// Themes
+:root,
+.mc-theme-dark {
+  // Main colors
+  --mc-theme-background:              #{$mc-color-dark};
+  --mc-theme-text:                    #{$mc-color-light};
+  --mc-theme-separator:               #{rgba($mc-color-light, 0.3)};
+  --mc-theme-border-contrast:         #{rgba($mc-color-light, 0.5)};
+  --mc-theme-border-contrast-hover:   #{$mc-color-light};
+  // Dropdown colors
+  --mc-theme-dropdown-backdrop:       #{rgba($mc-color-dark, 0.9)};
+  --mc-theme-dropdown-border:         #{$mc-color-gray-200};
+  --mc-theme-dropdown-cont-bg:        #{$mc-color-gray-100};
```

since there's not really any other backgrounds for the dropdown this could conflict with, let's drop the `-cont`
```suggestion
  --mc-theme-dropdown-bg:        #{$mc-color-gray-100};
```